sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -330,6 +365,11 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
     calculateKey(Bytes.concat("receipt".getBytes, txHash))
   }
 
+  private[horizen] def getTransactionBlockNumberKey(txId : scorex.util.ModifierId): ByteArrayWrapper = {
+    calculateKey(Bytes.concat("txblock".getBytes, txId.getBytes))
```

Comment:
Let's do `idToBytes(txId)` instead of `txId.getBytes`

Reply:
Done